src/DisplayFormats/include/DisplayFormats/DisplayFormats.h

```diff
@@ -11,6 +11,10 @@ namespace orbit_display_formats {
 
 [[nodiscard]] std::string GetDisplaySize(uint64_t size_bytes);
 
+enum class TimeUnit { kNanosecond, kMicrosecod, kMillisecond, kSecond, kMinute, kHour, kDay };
+std::string DisplayTimeUnit(TimeUnit unit);
+TimeUnit ChooseUnitForDisplayTime(absl::Duration duration);
+double ToDoubleTimeUnits(absl::Duration duration, TimeUnit unit);
```

Comment:
I assume this function is named like the abseil functions `ToDoubleNanoseconds`, right? While for the abseil functions with the given unit it's clear to me what it means, it's not clear to me what `ToDoubleTimeUnits` when I see this somewhere in the code. I'm wondering what could be a clearer name. Maybe `ToDoubleInGivenTimeUnit`? (I don't feel strongly about this. If we don't find a better name, feel free to leave it.)